### PR TITLE
3_part_without_review

### PR DESCRIPTION
**Pull request by YaTut1901**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Sergey
+  Date: 08.09.2020
+  Time: 15:49
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (dariiabutorina) [inline src/main/webapp/WEB-INF/views/index.jsp]: I find it confusing that you're Sergey now, Ivan... So let's remove that

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>main.internet.shop.controller.IndexController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (dariiabutorina) [inline src/main/webapp/WEB-INF/web.xml]: Blank line!

**Review by dariiabutorina — CHANGES_REQUESTED** [review]
Generics?..

**Review by dariiabutorina — APPROVED** [review]

**Review by RomanDubovskyi — COMMENTED** [review]
Some files from hw is missing, for example where is GenericDao class in PR ?

**Comment by RomanDubovskyi** [comment]
insert new link please

**Pull request closed without merging**